src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -33,6 +33,7 @@ class Yaml
     const PARSE_CONSTANT = 256;
     const PARSE_CUSTOM_TAGS = 512;
     const DUMP_EMPTY_ARRAY_AS_SEQUENCE = 1024;
+    const DUMP_NULL_AS_TILDE = 2048;
```

Comment:
Is TILDE correct in English or shall I rename the constant?

Reply:
According to wikipedia it seems that the correct word: https://en.wikipedia.org/wiki/Tilde